pkg/errors/errors.go

```diff
@@ -0,0 +1,8 @@
+package errors
+
+import "errors"
+
+var (
+	NoSupportedVersionAvailableErr = errors.New("no supported version available")
+	VersionNotAvailableErr         = errors.New("version not available")
+)
```

Comment:
Can't we find a more well suited place for this file to be placed in? I'm hesitant with such files, as they just introduce more little helper packages. Either this error is returned by a package and also in that package, or it is consumed by a package and inside that package. 

For example `VersionNotAvailableErr` is only used in the `package ubuntu` so place it there.

Reply:
fixed in 3e8b884